addons/default/visiosoft/advs-module/src/Adv/AdvRepository.php

```diff
@@ -58,7 +58,7 @@ public function searchAdvs(
                 $delimiter = '_';
                 $keyword = str_slug($param['keyword'], $delimiter);
                 $query = $query->where(function ($query) use ($keyword) {
-                    $query->where('advs_advs_translations.advs_desc', 'like', '%' . $keyword . '%')
+                    $query->where('advs_advs_translations.advs_desc', 'like', $keyword)
```

Comment:
that will not work for long descriptions, maybe try a different method

Reply:
searhing desc is not making sense espacially from mysql database. maybe we can remove it? @spektra2147 @muammertop 
@diashalabi